adminSiteServer/apiRouter.ts

```diff
@@ -3283,6 +3287,44 @@ getRouteWithROTransaction(
     }
 )
 
+getRouteWithROTransaction(
+    apiRouter,
+    `/gpt/suggest-alt-text/:imageId`,
+    async (
+        req: Request,
+        res,
+        trx
+    ): Promise<{
+        success: boolean
+        altText: string | null
+    }> => {
+        const imageId = parseIntOrUndefined(req.params.imageId)
+        if (!imageId) throw new JsonError(`Invalid image ID`, 400)
+        const image = await trx<DbEnrichedImage>("images")
+            .where("id", imageId)
+            .first()
+        if (!image) throw new JsonError(`No image found for ID ${imageId}`, 404)
+
+        const src = `${CLOUDFLARE_IMAGES_URL}/${image.cloudflareId}/public`
+        let altText: string | null = ""
+        try {
+            altText = await fetchGptGeneratedAltText(src)
+        } catch (error) {
+            console.error(
+                `Error fetching GPT alt text for image ${imageId}`,
+                error
+            )
```

Comment:
Out of scope of this PR, but it's unfortunate if we currently have to effectively log the error twice to surface/log it properly. This shouldn't be necessary, in general.